websites/T/Taiyō/metadata.json

```
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"name": "Luscarvalho",
+		"id": "458776046184169473"
+	},
+	"service": "Taiyō",
+	"altnames": [
+		"Taiyō.moe"
+	],
+	"description": {
+		"en": "Read manga online for free at Taiyō, without ads, with high-quality images, and reader support.",
+		"pt_BR": "Leia mangás online e gratuitamente na Taiyō, sem anúncios, com alta qualidade de imagens e suporte aos leitores."
+	},
+	"url": "taiyo.moe",
```

Site supports `www` subdomain.
```suggestion
	"url": [
		"taiyo.moe",
		"www.taiyo.moe"
	],
```